src/zenml/cli/secret.py

```diff
@@ -98,13 +98,15 @@ def register_secret(
 
     Args:
         secrets_manager: Stack component that implements the interface to the
-            underlying secrets engine
+        underlying secrets engine
```

Comment:
Not sure if it affects our API docs, but personally I prefer these indented to more easily differentiate between the different attributes

Reply:
Basically I was trying to fix the cli help message, but that one is broken either way:

```shell
zenml secret register --help
```

```
Usage: zenml secret register [OPTIONS] NAME

  Register a secret with the given name as key

  Args:     secrets_manager: Stack component that implements the interface
  to the     underlying secrets engine     name: Name of the secret
  secret_schema_type: Type of the secret schema - make sure the schema of
  choice is registered with the secret_schema_class_registry     secret_key:
  Key of the secret key-value pair     secret_value: Value of the secret
  Key-value pair

Options:
  -s, --schema TEXT  Register a secret with an optional schema.
  -k, --key TEXT     The key to associate with the secret.
  -v, --value TEXT   The value to associate with the secret key.
  --help             Show this message and exit.

```

Reply:
I'll re-indent - any ideas for the cli help formatting in the meantime?